### ISSUE TITLE
Cannot INSERT negative numbers without quotes

### ISSUE THREAD

**Issue by apavlo** [cmu-db/peloton#518]
You cannot INSERT values with minus sign in front of them:
```
postgres=# CREATE TABLE foo (val INTEGER);
CREATE TABLE foo (val INTEGER) 0
postgres=# INSERT INTO foo VALUES (-100);
Error parsing SQL statement
```

Unless you wrap them in quotes:
```
postgres=# INSERT INTO foo VALUES ('-100');
INSERT INTO foo VALUES ('-100') 1
postgres=# SELECT * FROM foo;
 val  
------
 -100
(1 row)
```

**Comment by raghavaggsss** [comment]
Why is there a 0 at the end of the statement CREATE TABLE foo (val INTEGER) 0?

**Comment by seojungmin** [comment]
I just copy my quick fix in sql_scanner.l. (Handling a negative FLOAT value should also be added later) As we know, this is going to be fixed permanently from #521.

diff --git a/src/parser/sql_scanner.l b/src/parser/sql_scanner.l
index 6e4e09f..3f95539 100644
--- a/src/parser/sql_scanner.l
+++ b/src/parser/sql_scanner.l
@@ -215,7 +215,7 @@ BWTREE              TOKEN(BWTREE)
        return SQL_FLOATVAL;
 }
 
-[0-9]+ {
+\-?[0-9]+      {
        yylval->ival = atol(yytext);
        return SQL_INTVAL;
 }

**Comment by apavlo** [comment]
@seojungmin Can you please send a PR?

**Comment by malin1993ml** [comment]
This has been fixed by #550.
